### DISCUSSION TITLE
How to resume training in Lightning Flash?

### DISCUSSION THREAD

**Discussion by eric-tc** [PyTorchLightning/lightning-flash#1322] [Q&A]
Hi I would like to know what is the best way to resume training in lightning flash? At the moment i am using load_from_checkpoint to start the training from my last saved model. Is there a better way to to this. For example in PL you have a parameter in trainer.fit(ckpt_path="...") with lightning flash i have not seen this option

**Comment by mkwisniewski** [comment]
You can try using `pytorch_lightning.Trainer.fit()` directly with `lightning-flash`'s model.
